array-destructuring/exercise-1/exercise.js

```diff
@@ -5,6 +5,7 @@ const personOne = {
 };
 
 function introduceYourself(___________________________) {
+  let {name,age,favouriteFood}= personOne;
```

Comment:
This is object destructuring correctly - good stuff.

See if you can enable Prettier in your VS Code - you can add the Prettier extension, then look in Settings for both the Default Formatter (set to "Prettier") and also make sure Format On Save is checked. It'll make your code look a lot better without you needing to do anything :)